addon/mixins/axis-titles.js

```diff
@@ -126,17 +156,19 @@ const AxisTitlesMixin = Ember.Mixin.create({
    * Position of x axis title on the y axis
    * @type {Number}
    */
-  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
-    return this.get('graphicBottom') + this.get('axisPadding');
+  xAxisPositionY: Ember.computed('graphicBottom', 'labelHeightOffset',
```

Comment:
I would prefer keeping as this for consistency with other function and easier to make changes.